### DISCUSSION TITLE
Action icon/button Option for Text Input Interface

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#6551] [Feature Requests]
Depending on the context of the field content, you may want to perform a common action for the data. For example, if the input is storing an email, phone number, or web link, you may want to have a "Launch" icon/button to simply open that in a new tab. I imagine this is an "Action" dropdown in the Interface Options, where we can add a list of common actions. This seems better than trying to "sniff" the data and add it automatically.

Would love to hear other options for this!

**Comment by philleepflorence** [comment]
This is an actual User Story that will benefit from this feature:

> As a content contributor, when adding and editing items within the Admin App, I want to be able to click on the options icon next to the URL interface label, in order to preview, confirm, or open the hyperlink in a browser.

- I can see multiple use cases for enabling actions (_preview in..., open in..., validate link, get link details..., copy related item ID..._) that can be configured within the interface options...potentially consolidating some custom interface extensions
- On the item details view, do you see this added to the current interface options dropdown (_reset..., undo..., raw value_) or a stand alone?

**Reply by benhaynes** [reply]
Standalone option for actions and/or opening items. If there are no other "actions" then it can just be an "open content" option toggle.

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
